client/jetpack-connect/style.scss

```diff
@@ -622,8 +622,8 @@
 	}
 
 	&.is-primary {
-		background-color: $green-jetpack;
-		border-color: darken( $green-jetpack, 5% );
+		background-color: var( --color-jetpack );
+		border-color: var( --color-jetpack-light );
```

Comment:
My concern with changing these lights to dark would mean that the hover state wouldn't have a visible effect. Is it okay to lose that?

Reply:
I've switched all the ones that you recommended to be dark.

> You could add (new) hover styles to work around this

So I understand correctly here, do you suggest adding a new variable, perhaps `--color-jetpack-hover`, so that there's a difference in hover styles currently?

	.button.is-primary {
		background-color: var( --color-jetpack );
		border-color: var( --color-jetpack-dark );

		&:hover,
		&:focus {
			border-color: var( --color-jetpack-dark );
		}`

Just to clarify, my concern is that the hover border will be identical. From a really brief look, I don't think there's too a major issue with background colours on hover though.

> In the long term we should use the actual <Button /> component and add Jetpack styles (is-jetpack?) there.

Agreed, happy to take a stab at that after this!

So to just clarify, is this correct?

**Background colours:**

Normal state - Jetpack green
Hover/Focus state - Lighter Jetpack 

**Border colours:**

Normal state - Dark Jetpack
Hover/Focus state - This PR currently proposes also Dark Jetpack, which seems off...but I think changing that would require a new variable. Perhaps `--color-jetpack-hover` at `#002508` which seems to be the one now, but I'm not certain about this.

For reference, here's the current styles. 

![fgdsfgdsfdsgfdg](https://user-images.githubusercontent.com/43215253/51625998-7ac5f700-1f36-11e9-82ad-a271bc5d7db3.png)

Reply:
My suggestion is to align it with the current `<Button />` implementation. Though we might need to introduce another light green color for hover states.

That's what the current `<Button />` for e.g. a primary one looks like in terms of colors:

- normal:
  - background: `--color-accent`
  - border: `--color-accent-dark`
- hover:
  - background: `--color-accent-400` (this one does not currently exist for Jetpack)
  - border: `--color-accent-dark` (border color doesn't change for hover)

/cc @drw158 

I hope that makes it more clear. Let me know if you have further questions and I'll try to answer them accordingly.

Reply:
I think I understand what you mean, I've just added two commits to try and address that. Let me know if that's what you're referring to! :)

Reply:
400 in Jetpack Green would be #57ca53

Let me know if you need any more!

Reply:
So currently, `--color-jetpack-light` is being used five times, and only for hover or focus button styles, either as the `background-color` or the `box-shadow`. In that case, should the current `#81d677` be replaced with `#57ca53`?

Reply:
Thanks for confirming, just added some commits, let me know if that's what you're referring to. :)